### ISSUE TITLE
Node 5 not working, used master copy.

### ISSUE THREAD

**Issue by robertbaker** [sass/node-sass#1239]
```
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (c:\git\pos\node_modules\node-gyp\lib\build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Windows_NT 10.0.10576
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "c:\\git\\pos\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
gyp ERR! cwd c:\git\pos\node_modules\node-sass
gyp ERR! node -v v5.0.0
gyp ERR! node-gyp -v v3.0.3
gyp ERR! not ok
Build failed

npm ERR! Windows_NT 10.0.10576
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "rebuild" "node-sass"
npm ERR! node v5.0.0
npm ERR! npm  v3.3.12
npm ERR! code ELIFECYCLE
npm ERR! node-sass@3.4.1 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the node-sass@3.4.1 postinstall script 'node scripts/build.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the node-sass package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node scripts/build.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls node-sass
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     c:\git\pos\npm-debug.log
```

**Comment by xzyfer** [comment]
Please search before opening new issue. We have node 5 compatibility in 3.4.1 but the tool we use to build windows binaries hasn't been updated yet.

The windows binaries will be back ported to 3.4.1 when they're available.

**Comment by jrock2004** [comment]
Is there a place we can watch to see when the 3.4.1 backport for windows is done?

**Comment by robertbaker** [comment]
I would leave this open. I did a search before I made this issue but the results I read say it works with node 5. By default the search only searches open issues.

**Comment by xzyfer** [comment]
Windows Node 5 binary is now available for v3.4.1 **only**. Just npm install.
